java/operator/org.eclipse.theia.cloud.operator/src/main/java/org/eclipse/theia/cloud/operator/handler/session/EagerStartSessionHandler.java

```diff
@@ -115,6 +118,25 @@ public boolean sessionAdded(Session session, String correlationId) {
             return false;
         }
 
+        try {
+            client.services().inNamespace(client.namespace()).withName(serviceToUse.get().getMetadata().getName())
+                .edit(service -> {
+                    LOGGER.info("Setting pod labels");
```

Comment:
I think this should only be a debug level log as nothing exiting or too relevant happens here functionality wise.

```suggestion
                    LOGGER.debug("Setting service labels");
```